pom.xml

```diff
@@ -323,11 +323,24 @@
             </exclusions>
         </dependency>
 
+        <!--JDBC driver for building connection with Snowflake-->
+        <dependency>
+            <groupId>net.snowflake</groupId>
+            <artifactId>snowflake-jdbc</artifactId>
+            <version>3.13.25</version>
```

Comment:
What's the reason of using 3.13.25 instead of other high versions? Do you know which version other SDKs/connectors are using?

Reply:
https://snowflake.slack.com/archives/C04URTJMC77/p1680152983458869

Reply:
On top of that
My philosophy:
1. We have seen issues in past by upgrading to the latest versions. 
2. The lesser the code changes lesser the probability of bugs. 3.13.25 is the min version we will have to support, so lets go with that. 


But again, I am also okay with upgrading to the latest and greatest.